server/controllers/errorController.js

```diff
@@ -0,0 +1,168 @@
+const cmd = require('node-cmd');
+const Error = require('../models/errorsModel')
+
+const HEADERS = [
+  'NAMESPACE',
+  'LAST SEEN',
+  'TYPE',
+  'REASON',
+  'OBJECT',
+  'MESSAGE'
+];
+
+const errorController = {};
+
+//GOAL: Get output from kubectl command, possibly send to another middleware for formatting and processing data, store in mongodb
+// --field-selector type=Failure
+//Get output from kubectl command and store in a array with each row being contained in each index
+errorController.queryErrors = async (req, res, next) => {
+  try {
+    const errorList = cmd.runSync('kubectl get events --all-namespaces').data.split('\n');
+    console.log(errorList[0].length, errorList[1].length, errorList[2].length)
+    //Pop the last row since it is empty
+    errorList.pop();
+    //Shift the first row since it is the headers
+    res.locals.errorList = errorList;
+    return next();
+  } catch (err) {
+    return next({
+      log: `errorController.queryErrors: ERROR: ${err}`,
+      message: {
+        err: 'Error occurred while querying errors. Check server logs for more information.',
+      }
+    })
+  }
+};
+
+//Store an array of the index of where each header in HEADERS array begins as well as the array length
+function headersIndex(array) {
+  let arr = [];
+  for (let i = 0; i < HEADERS.length; i++) {
+    arr.push(array.indexOf(HEADERS[i]));
+  }
+  //arr.push(array.length);
+  console.log('headersIndex', arr);
+  return arr;
+}
+
+//Converts ErrorList array of strings to nested array
+function errorArrayConverter(array) {
```

Comment:
Move this function declaration to line 12 and convert to arrow function (assuming that doesn't provoke an error).